### ISSUE TITLE
'week' scale is missing from TimelineTimeAxisScaleType (vis-timeline/types/index.d.ts)

### ISSUE THREAD

**Issue by niksonkjohn** [visjs/vis-timeline#1156]
'week' scale is missing from TimelineTimeAxisScaleType.
File: vis-timeline/types/index.d.ts

```
export type TimelineTimeAxisScaleType = 'millisecond' | 'second' | 'minute' | 'hour' |
  'weekday' | 'day' | 'month' | 'year';
```

We are using week scale in our timeline. can you please add 'week' scale to the scale type

**Comment by mojoaxel** [comment]
@niksonkjohn pull-requests are always welcome :wink:

**Comment by gucolin** [comment]
Having the same issue here. Any update for this?

**Comment by mojoaxel** [comment]
(finally) fixed with #1426
